cognite/neat/_rules/exporters/_rules2excel.py

```diff
@@ -196,199 +211,238 @@ def _hide_internal_columns(self, workbook: Workbook) -> None:
+            workbook,
+            sheet_name="Properties",
+            column_name="View" if role == RoleTypes.dms else "Class",
+            data_validator_name="views_or_classes",
+            data_validators=data_validators,
+            no_validation_values=100 * 100,
+            no_rows=100,
+        )
 
-        if column := find_column_with_value(workbook["Properties"], "Value Type"):
-            dv_value_types.add(f"{column}{3}:{column}{no_rows * 100}")
+        if role == RoleTypes.dms:
+            self._add_data_validation(
+                workbook,
+                sheet_name="Properties",
+                column_name="Container",
+                data_validator_name="container",
+                data_validators=data_validators,
+                no_validation_values=100,
+                no_rows=100 * 100,
+            )
 
-        if column := find_column_with_value(workbook["Classes"], "Implements"):
-            dv_implements.add(f"{column}{3}:{column}{no_rows}")
+            self._add_data_validation(
+                workbook,
+                sheet_name="Properties",
+                column_name="Immutable",
+                data_validator_name="immutable",
+                data_validators=data_validators,
+                no_validation_values=2,
+                no_rows=100 * 100,
+            )
+
+            self._add_data_validation(
+                workbook,
+                sheet_name="Views",
+                column_name="In Model",
+                data_validator_name="in_model",
+                data_validators=data_validators,
+                no_validation_values=2,
+                no_rows=100,  # 100 views
+            )
+
+            self._add_data_validation(
+                workbook,
+                sheet_name="Containers",
+                column_name="Used For",
+                data_validator_name="used_for",
+                data_validators=data_validators,
+                no_validation_values=3,
+                no_rows=100,  # 100 views
+            )
+
+    def _make_helper_sheet(
+        self,
+        role: RoleTypes,
+        workbook: Workbook,
+        no_rows: int,
+    ) -> None:
+        """This helper sheet is used as source of data for drop down menus creation
+
+        Args:
+            role: The role for which the helper sheet is created. Can be either "dms" or "information".
+            workbook: Workbook representation of the Excel file.
+            no_rows: number of rows to add data too that will form base for drop down menus.
+        """
 
-    def _make_helper_info_sheet(self, workbook: Workbook, no_rows: int) -> None:
-        """This helper Information sheet is used as source of data for drop down menus creation"""
         workbook.create_sheet(title=self._helper_sheet_name)
 
-        for dtype_counter, dtype in enumerate(_DATA_TYPE_BY_DMS_TYPE.values()):
+        for value_type_counter, value_type in enumerate(_DATA_TYPE_BY_DMS_TYPE.values()):
```

Comment:
Nitpick. If the loop does not execute below, then you will get an error on line 343 `value_type_counter += 1` unless you have this line here.
```suggestion
        value_type_counter = 0

        for value_type_counter, value_type in enumerate(_DATA_TYPE_BY_DMS_TYPE.values()):
```